how-to-contribute.md

```diff
@@ -0,0 +1,65 @@
+# How to contribute to the Knowledge Hub
+
+We are building the Knowledge Hub in public and we would like to encourage contributions from the community. 
+
+The only thing you need to contribute to the Knowledge Hub is a Github account. You can sign up to GitHub by filling out the form [on this page](https://github.com/join).
```

Comment:
To suggest changes, you'll need a GitHub account. You can sign up [on this page](https://github.com/join). It only takes a couple of minutes.